pkg/kubelet/kubelet_node_status.go

```diff
@@ -704,3 +735,17 @@ func nodeConditionsHaveChanged(originalConditions []v1.NodeCondition, conditions
 	}
 	return false
 }
+
+// providerRequiresNetworkingConfiguration returns whether the cloud provider
+// requires special networking configuration.
+func providerRequiresNetworkingConfiguration(cloud cloudprovider.Interface) bool {
```

Comment:
I wonder why node_status is a better place for this method than kubelet_network.go? This method seems to be network related?

Reply:
These are interim-status, actually all node status logic should local in kubelet/nodestatus, I had done it but derek thought it's too large to review.